code/game/machinery/cloning.dm

```diff
@@ -163,15 +163,11 @@
 
 	H.hardset_dna(ui, se, H.real_name, null, mrace, features)
 
-	if(efficiency > 2)
-		var/list/unclean_mutations = (GLOB.not_good_mutations|GLOB.bad_mutations)
-		H.dna.remove_mutation_group(unclean_mutations)
-	if(efficiency > 5 && prob(20))
-		H.randmutvg()
-	if(efficiency < 3 && prob(50))
-		var/mob/M = H.randmutb()
-		if(ismob(M))
-			H = M
+	if(prob(50 - efficiency*10)) //Chance to give a bad mutation.
+		H.randmutb() //100% bad mutation. Can be cured with mutadone.
+
+	if(prob(50 - efficiency*10)) //Chance to scramble your DNA
+		scramble_dna(H,ui=TRUE,10) //Chance to scramble your visual DNA a little, with 10% occuring for each node.

```

Comment:
DNA already gets scrambled not matter what.
This can lead to ling checking/clucking

Reply:
Where does it get scrambled?

Reply:
Even max level mess up DNA stands, changing things so that if you have DNA locked gear then it will lock you out.
Thats why clone disks dont store a copy of your cloning data it stores DNA